src/libsystemd-network/sd-dhcp6-client.c

```diff
@@ -59,6 +61,7 @@ struct sd_dhcp6_client {
         be16_t *req_opts;
         size_t req_opts_allocated;
         size_t req_opts_len;
+        char *fqdn;
```

Comment:
afaics this is leaked right now when sd_dhcp6_client is leaked

Reply:
Agreed, this needs freeing.